src/utils/loadScripts.py

```diff
@@ -1,180 +1,362 @@
 ########### Import Libraries
 import logging, verboselogs
+
 logger = verboselogs.VerboseLogger(__name__)
 from dateutil.relativedelta import relativedelta as rd
-import requests 
-from src.utils.rest import rest_get, rest_post # 
+import requests
+from src.utils.rest import rest_get, rest_post  #
 import asyncio
 from packaging import version
 
+import logging
+from datetime import datetime
+import pytz  # Requires installing the pytz module
```

Comment:
If we are correctly changing the system time of the container that runs this script (as suggested above), then I think we don't need to use pytz to change the timezone again within the python script, and we can simply rely on the system time. I think much of this code is the not needed

Reply:
While I agree that creating a custom `logging` class just to incorporate timestamps does involve more lines than one would think, the additions to the `Dockerfile` from the comment above involving `tzdata` is also quite _ugly_ and also faces the same length issue as the implementation here.

Reply:
If we changed to alpine, it take it the addition would be very small on the .yml files. Would you agree we then don't need a custom class?

Reply:
I agree we dont need a custom class, however see my comment above about changing to alpine